### ISSUE TITLE
Sending emails with unicode (e.g. æ, ø, å) fails

### ISSUE THREAD

**Issue by jsfr** [sendgrid/sendgrid-nodejs#242]
#### Issue Summary

Trying to send an email using containing a unicode char such as æ, ø, å or € in the htmlContent results in an error with status 400 as the below response

```
{ statusCode: 400,
  body: '{"errors":[{"message":"Bad Request","field":null,"help":null}]}',
  headers:
   { server: 'nginx',
     date: 'Thu, 16 Jun 2016 13:20:56 GMT',
     'content-type': 'application/json',
     'content-length': '63',
     connection: 'close',
     'x-frame-options': 'DENY' } }
```

#### Steps to Reproduce

1. Send email using the helper with a unicode char in the htmlContent e.g. æ, ø, å or €

#### Technical details:

* sendgrid-nodejs Version: 3.0.4
* Node.js Version: 6.2.1

**Comment by jsfr** [comment]
Okay it seems this is fixed as of 3.0.5 (possibly related to #239 ?) so I'm closing it again

**Comment by thinkingserious** [comment]
@jsfr you are correct, version 3.0.5 fixed this problem. Thanks!
